src/Tizen.NET.MaterialComponents/MaterialComponents.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public enum ComponentsOptions
```

Comment:
Make ComponentsOptions a `class` or `struct` for sensibility, then add properties.

Reply:
Plus, I think `InitializationOptions` is also OK

Reply:
Fixed.